packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -1,99 +1,137 @@
-import { OnDemandPageScanReport, OnDemandPageScanResult, OnDemandPageScanRunResult, OnDemandScanResult } from 'storage-documents';
+import { OnDemandPageScanRunResultProvider, PageScanRunReportService } from 'service-library';
+import {
+    OnDemandPageScanReport,
+    OnDemandPageScanResult,
+    OnDemandPageScanRunResult,
+    OnDemandPageScanRunState,
+    OnDemandScanResult,
+    ScanState,
+} from 'storage-documents';
 import { ScanMetadataConfig } from '../scan-metadata-config';
 import { ScannerTask } from '../tasks/scanner-task';
 import { WebDriverTask } from '../tasks/web-driver-task';
 
 @injectable()
 export class Runner {
     constructor(
+        @inject(GuidGenerator) private readonly guidGenerator: GuidGenerator,
         @inject(ScanMetadataConfig) private readonly scanMetadataConfig: ScanMetadataConfig,
         @inject(ScannerTask) private readonly scannerTask: ScannerTask,
         @inject(OnDemandPageScanRunResultProvider) private readonly onDemandPageScanRunResultProvider: OnDemandPageScanRunResultProvider,
         @inject(WebDriverTask) private readonly webDriverTask: WebDriverTask,
         @inject(Logger) private readonly logger: Logger,
+        @inject(PageScanRunReportService) private readonly pageScanRunReportService: PageScanRunReportService,
     ) {}
 
     public async run(): Promise<void> {
         let browser: Browser;
         const scanMetadata = this.scanMetadataConfig.getConfig();
-        let onDemandScanResult: OnDemandScanResult;
-        let onDemandPageScanReport: OnDemandPageScanReport;
-        let onDemandPageScanRunResult: OnDemandPageScanRunResult;
-        let pageScanRunResult: OnDemandPageScanResult;
-        let axeScanResults: AxeScanResults;
+        let pageScanResult: OnDemandPageScanResult;
 
-        try {
-            this.logger.logInfo(`Reading Page Scan Run id= ${scanMetadata.id}`);
-            pageScanRunResult = (await this.onDemandPageScanRunResultProvider.readScanRuns([scanMetadata.id]))[0];
-
-            onDemandScanResult = {
-                state: 'unknown',
-                issueCount: undefined,
-            };
-
-            onDemandPageScanReport = {
-                reportId: '',
-                format: 'sarif',
-                href: '',
-            };
-
-            // set scanned page run state to running
-            onDemandPageScanRunResult = {
-                state: 'running',
-                timestamp: new Date().toJSON(),
-                error: '',
-            };
-
-            pageScanRunResult.run = onDemandPageScanRunResult;
-
-            this.logger.logInfo(`Changing page status to running`);
-            this.logger.logInfo(`Updating page scan in database`);
-            await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanRunResult);
-            pageScanRunResult = (await this.onDemandPageScanRunResultProvider.readScanRuns([scanMetadata.id]))[0];
+        this.logger.logInfo(`Reading Page Scan Run id= ${scanMetadata.id}`);
```

Comment:
We need either remove id from log message or add it everywhere in file using logger options param. This logs w/o id only useful on console in batch/dev

Reply:
this is not something we have done in job manager. so, i would consider this as improvement & do it in next prs. We can anyway debug & get related logs for this line, by finding all logs that belong to this task id.